### PR TITLE
Added support for running with Docker

### PR DESCRIPTION
**Pull request by kennethkalmer**
There might be users that want to use this project, but they lack access to
a suitable Ruby environment. I think running via docker and discarding the
container afterwards is much easier than expecting a fully functional Ruby
environment.

I also included 7fb987b in this one.

I'll gladly help with explanations for publishing your own automated build if
this gets accepted.

### CODE REVIEW

**Comment by kennethkalmer** [comment]
I've since tweaked things a bit more, and this might not be suitable as a PR anymore, but I'm leaving it open for discovery and feedback.

Basically I wanted the newest `ruby-trello` gem, but that required a `rest-client` version that wasn't compatible with the pivotal client. I then switch to another fork of the pivotal client that updated the dependencies, and all seems well.

I also needed to add `git-core` to the Dockerfile due to the Gemfile pointing to a Github repo.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Prepare
+RUN mkdir /app
+COPY . /app
+
+# Install lean
+RUN cd /app && bundle install --without development
+
+# And run
+WORKDIR /app
+ENTRYPOINT ["bundle exec /app/bin/pivotal-to-trello"]
```
Comment (rouge8) [inline Dockerfile:24]: This breaks in recent version of Docker. Needs to be:

```
ENTRYPOINT ["bundle", "exec", "/app/bin/pivotal-to-trello"]
```

**Comment by rouge8** [comment]
With the `ENTRYPOINT` fix above, I get this error when I try and run it:

```
The gemspec at /var/lib/gems/2.1.0/bundler/gems/pivotal-tracker-9c038f25dc90/pivotal-tracker.gemspec is not valid. The validation error was 'duplicate dependency on rest-client (>= 1.7.2), (>= 1.7.2) use:
    add_runtime_dependency 'rest-client', '>= 1.7.2', '>= 1.7.2'
'
```

**Comment by kennethkalmer** [comment]
Thanks for the feedback, will have a look!

**Comment by recurser** [comment]
Thanks @kennethkalmer - apologies for the huge delay. I've continued this here and made some minor tweaks: https://github.com/recurser/pivotal-to-trello/pull/14

**Pull request closed without merging**